graph/gaedatastore/iterator.go

```diff
@@ -171,7 +176,12 @@ func newIteratorNext(qs *QuadStore, k string, d quad.Direction, t *Token) *itera
 		clog.Errorf("Cannot create iterator without a valid context")
 		return &iteratorNext{done: true}
 	}
-	name := quad.StringOf(qs.NameOf(t))
+	tn, err := qs.NameOf(t)
+	if err != nil {
+		clog.Errorf("Creating iterator token lookup error: %v", err)
+		return &iteratorNext{done: true}
```

Comment:
```suggestion
		return &iteratorNext{done: true, err: err}
```

Reply:
Are you sure? Given that there are a lot of other locations in the file where errors return done: true without the error field?

Reply:
I think that might be a copy-paste error or something. Or it might not check for NotFound errors properly.